### PR TITLE
use latest isaac sim for tutorial

### PR DESCRIPTION
**Pull request by knorth55**
### Description

This PR change to search installed `isaac-sim` and use latest isaac-sim`.

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit2/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Review by JafarAbdi — APPROVED** [review]

File: doc/how_to_guides/isaac_panda/launch/python.sh [hunk 1]
```diff
@@ -18,6 +18,6 @@ do
     NEW_ARGS="${NEW_ARGS} ${CUR_SCRIPT_DIR}/${arg}"
 done
 
-pushd ${ISAAC_SCRIPT_DIR}
+pushd ${ISAAC_SCRIPT_DIRS[${#ISAAC_SCRIPT_DIRS[@]}-1]}
```
Comment (MarqRazz) [inline doc/how_to_guides/isaac_panda/launch/python.sh:21]: This does mean that someone will have to manually check that ALL released versions of Isaac, now and into the future, are compatible with this tutorials omnigraph creation script. Pretty sure this is safe right now but Nvidia has been known to make breaking changes to that API. What do you think about adding some checks to verify the version against a known working list?

Comment (knorth55) [reply doc/how_to_guides/isaac_panda/launch/python.sh:21]: Hmm, your suggestion is right.
I don't trust nvidia either.
I checked this working in `2022.2.1`, so we should check if the package is `2022.2.1` or `2022.2.0`.

Comment (MarqRazz) [reply doc/how_to_guides/isaac_panda/launch/python.sh:21]: I have also run this back to `1.1` but would have to test it again to make sure it still works. I'm okay with this launch script only supporting `2.0` and `2.1` for now.

Comment (knorth55) [reply doc/how_to_guides/isaac_panda/launch/python.sh:29]: Thank you for your information. I updated.

**Pull request merged**